README.md

```diff
@@ -65,6 +65,40 @@ High performance and convenient features are the main goals.
     cargo install --locked --git https://github.com/pamburus/hl.git
     ```
 
+* Run by using [nix](https://nixos.org/download/)
+
+  ```sh
+  nix run github:pamburus/hl
+  ```
+
+* Install the package by using [nix-flakes](https://wiki.nixos.org/wiki/Flakes)
+
+  ```nix
```

Comment:
This example looks quite verbose, could you please hide it under a [collapsed section](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections)?
For example, like this
````markdown
  <details>
  <summary>Example how to update nix configuration</summary>

  ```nix
  {
    ...
  }
  ```

  </details>
````